packages/fast-form-generator-react/src/form/form-item.children.style.ts

```diff
@@ -24,36 +24,29 @@ const styles: ComponentStyles<FormItemChildrenClassNameContract, {}> = {
         position: "relative",
         display: "flex",
         flexDirection: "column",
+        ...applyControlWrapper(),
     },
     formItemChildren_control: {
         ...applyControl(),
         verticalAlign: "middle",
     },
     formItemChildren_controlLabel: {
         ...applyLabelStyle(),
-        display: "block",
-        marginTop: toPx(12),
     },
     formItemChildren_existingChildren: {
         ...applyCleanListStyle(),
     },
+    formItemChildren_existingChildren__sorting: {
+        fontSize: "11px",
+        color: foreground300,
+        backgroundColor: `${background100} !important`,
```

Comment:
you should never have to use !important here, instead inside the `formItemChildren_existingChildren` key you can add:

```
"&.$formItemChildren_existingChildren__sorting" : {
      // your override requiring more specificity
}
```